Resources/Prototypes/Recipes/Lathes/implants.yml

```diff
@@ -0,0 +1,132 @@
+# Fun
+
+- type: latheRecipe
+  id: SadTromboneImplanter
+  icon:
+      sprite: Objects/Specific/Medical/implanter.rsi
+      state: implanter0
+  result: SadTromboneImplanter
+  completetime: 5
+  materials:
+    Plastic: 100
+    Glass: 150
+    Silver: 50
+
+- type: latheRecipe
+  id: LightImplanter
+  icon:
+      sprite: Objects/Specific/Medical/implanter.rsi
+      state: implanter0
+  result: LightImplanter
+  completetime: 5
+  materials:
+    Plastic: 100
+    Glass: 150
+    Silver: 50
+
+- type: latheRecipe
+  id: BikeHornImplanter
+  icon:
+      sprite: Objects/Specific/Medical/implanter.rsi
+      state: implanter0
+  result: BikeHornImplanter
+  completetime: 5
+  materials:
+    Plastic: 100
+    Glass: 150
+    Silver: 50
+
+# Sec
+
+- type: latheRecipe
+  id: TrackingImplanter
+  icon:
+      sprite: Objects/Specific/Medical/implanter.rsi
+      state: implanter0
+  result: TrackingImplanter
+  completetime: 5
+  materials:
+    Plastic: 100
+    Glass: 150
+    Silver: 100
+    Plasma: 50
+
+- type: latheRecipe
+  id: MindShieldImplanter
+  icon:
+      sprite: Objects/Specific/Medical/implanter.rsi
+      state: implanter0
+  result: MindShieldImplanter
+  completetime: 5
+  materials:
+    Plastic: 100
+    Glass: 150
+    Silver: 100
+    Plasma: 50
+
+- type: latheRecipe
+  id: ProtokineticImplanter
+  icon:
+      sprite: Objects/Specific/Medical/implanter.rsi
+      state: implanter0
+  result: ProtokineticImplanter
+  completetime: 5
+  materials:
+    Plastic: 100
+    Glass: 150
+    Silver: 100
+    Plasma: 150
+
+# Antag
+
```

Comment:
These shouldn't be printable, even by emag. It's huge powercreep and they're meant to be bought via uplink.